qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/ClusteringRule.java

```diff
@@ -288,39 +285,65 @@ private Topic waitForTopicAvailability(final String topicName) {
         .get();
   }
 
-  private Broker startBroker(final SocketAddress socketAddress, final String configFile) {
-    final File base = brokerBases.get(socketAddress);
+  private void startBroker(final int brokerId) {
+    if (brokerCfgs[brokerId] == null) {
+      brokerBases[brokerId] = Files.newTemporaryFolder();
+      closeables.add(() -> FileUtil.deleteFolder(brokerBases[brokerId].getAbsolutePath()));
+
+      final InputStream config =
+          this.getClass().getClassLoader().getResourceAsStream(brokerConfigFiles[brokerId]);
+      final BrokerCfg brokerCfg = TomlConfigurationReader.read(config);
+      EmbeddedBrokerRule.assignSocketAddresses(brokerCfg);
+
+      if (brokerCfgs[0] != null) {
+        brokerCfg
+            .getCluster()
+            .setInitialContactPoints(
+                new String[] {
+                  brokerCfgs[0].getNetwork().getManagement().toSocketAddress().toString()
```

Comment:
So they will join all the first node right?

Reply:
Yes with this change every node other then the first will always join the first node as it was before configured in the configuration files.